src/components/Modal/ConfirmDeleteModal.tsx

```diff
@@ -0,0 +1,66 @@
+import { commentDeleteState } from '@/store/recoilCommentState';
+import {
+  confirmDeleteCommentModalState,
+  confirmDeleteReviewModalState,
+} from '@/store/recoilModalState';
+import { reviewDeleteState } from '@/store/recoilReviewState';
+import { useRecoilState, useSetRecoilState } from 'recoil';
+import ModalPortal from './ModalPortal';
+
+type ConfirmDeleteModalType = {
+  target: string;
+  text: string;
+};
+
+const ConfirmDeleteModal = ({ target, text }: ConfirmDeleteModalType) => {
+  const [commentDeleteOpen, setCommentDeleteOpen] = useRecoilState<boolean>(
+    confirmDeleteCommentModalState
+  );
+  const [reviewDeleteOpen, setReviewDeleteOpen] = useRecoilState<boolean>(
+    confirmDeleteReviewModalState
+  );
+  const isDeleteComment = useSetRecoilState(commentDeleteState);
+  const isDeleteReview = useSetRecoilState(reviewDeleteState);
+
+  const onDeleteHandler = (target: string) => {
+    if (target === 'review') {
+      isDeleteReview(true);
+      setReviewDeleteOpen(false);
+    } else if (target === 'comment') {
```

Comment:
```suggestion
    } 
    if (target === 'comment') {
```

proposal;

if 문으로만 조건문을 만들면 저는 가독성이 올라가더라고요.